sh_stm: Fix USB overcurrent detect PIO on STi7141

Correct USB Overcurrent detect PIO matrix

Signed-off-by: Christopher TOMLINSON <christopher.tomlinson@st.com>
Index: linux/arch/sh/kernel/cpu/sh4/setup-stx7141.c
===================================================================
--- linux.orig/arch/sh/kernel/cpu/sh4/setup-stx7141.c	2008-08-06 17:59:40.000000000 +0100
+++ linux/arch/sh/kernel/cpu/sh4/setup-stx7141.c	2008-08-06 18:01:42.000000000 +0100
@@ -258,7 +258,7 @@
 		} pwr, oc;
 	} usb_pins[4] = {
 		{ { 4, 7, 1 }, { 4, 6, 1 } },
-		{ { 5, 1, 1 }, { 5, 2, 1 } },
+		{ { 5, 1, 1 }, { 5, 0, 1 } },
 		{ { 4, 3, 1 }, { 4, 2, 1 } },
 		{ { 4, 5, 1 }, { 4, 4, 1 } }
 	};
